RevenueCatUI/Templates/V2/PaywallsV2View.swift

```diff
@@ -361,23 +365,38 @@ fileprivate extension PaywallsV2View {
             return .init(locale: Locale.current, localizedStrings: PaywallComponent.LocalizationDictionary())
         }
 
+        var notFoundLocales = [Locale]()
```

Comment:
Won't this have at most one locale? 🤔 
What am I missing?

Reply:
Good catch! I need to add both the `chosenLocale` and the `defaultLocale` to the array in the final `else` clause below, and I only added the `defaultLocale` 🤦 
Thank you!

Reply:
Fixed in https://github.com/RevenueCat/purchases-ios/pull/5292/commits/19834155e0333f1dc4688487e2dae9c72598845f